sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,29 +1,41 @@
 # Release History
 
-## 2.0.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (2021-03-23)
 
 This release continues with the changes from `1.2.4` and `1.2.4-beta.1`.
```

Comment:
We don;t have to call out that v2 will have changes from v1.2.4. That is expected.
The call out for 1.2.4-beta.1 was added because the feature added there never went GA in any of the v1 releases.

I would recommend we simplify things and reduce confusion by removing this entire line and including the changelog from 1.2.4-beta.1 directly here for the 2.0.0-beta.1 release. The only thing there is about the auth flow and that is an important one and should get attention here